### DISCUSSION TITLE
Failed CSV Import to Firefly III via API

### DISCUSSION THREAD

**Discussion by Tweed-Bear** [firefly-iii/firefly-iii#9445] [Firefly III]
Hi everyone,

I'm currently working on an n8n workflow to automatically import all CSV files from a specific folder via the Firefly III API. Unfortunately, the imports aren't working as expected, even though the POST request is successful and the logs confirm the import has started. But nothing is imported to FireFly.

Output HTTP Node n8n:
`[
  {
    "data": "2024-11-08 14:25:57: Overrule token with token from Authorization header.\n2024-11-08 14:25:57: Trying to connect to http://192.168.178.48:5222...\n2024-11-08 14:25:57: The last 25 chars of the access token are: 9XC-F8[CENSORED 😊])7oSPK1pAk\n"
  }
]
`

FireFly III Log:

`firefly_iii_core         | 172.29.16.6 - - [08/Nov/2024:14:25:57 +0100] "GET /api/v1/about HTTP/1.1" 200 996 "-" "GuzzleHttp/7"
firefly_iii_importer     | [2024-11-08 14:25:58] local.INFO: Going to automatically import everything found in /import (/import)  
firefly_iii_importer     | 192.168.[X].[X] - - [08/Nov/2024:14:25:57 +0100] "POST /autoimport?directory=%2Fimport&secret=[CENSORED 😊] HTTP/1.1" 200 1396 "-" "axios/1.7.4"
`

I'm running Firefly with Docker on a NAS, and manual imports of the same files via the Firefly III- importer web interface work fine.

The folder contains both CSV and JSON files with identical names.

I'm a bit stumped as to what the issue might be. Perhaps someone here has had similar experiences or has an idea of what I could check

Thank you for your help!

+++Update 09.11. +++
Tried Postman as well to test the import. Unfortunately, with the same result.

Postman
Status: 200

`2024-11-09 08:40:37: Overrule token with token from Authorization header.
2024-11-09 08:40:37: Trying to connect to http://192.168.178.48:5222...
2024-11-09 08:40:37: The last 25 chars of the access token are: 9XC-[CENSORED 😊]7oSPK1pAk`

FireflyIII Log:

`firefly_iii_core         | 172.29.16.6 - - [09/Nov/2024:08:40:37 +0100] "GET /api/v1/about HTTP/1.1" 200 996 "-" "GuzzleHttp/7"
firefly_iii_importer     | [2024-11-09 08:40:37] local.INFO: Going to automatically import everything found in /import (/import)  
firefly_iii_importer     | 192.168.178.20 - - [09/Nov/2024:08:40:37 +0100] "POST /autoimport?directory=/import&secret= [CENSORED 😊] HTTP/1.1" 200 1396 "-" "PostmanRuntime/7.42.0"`

But nothing was imported to FireFly

**Comment by JC5** [comment]
Mmm, I see no continued logs. The data importer seems to log INFO only. Are there no other errors that you can find?

**Comment by Tweed-Bear** [comment]
Hi,
Apologies, I've maximized the logging level to 'debug'. Unfortunately, this is all the output I'm receiving after start and trigger the Import.

[20241109_Log_Fireflyiii.txt](https://github.com/user-attachments/files/17691693/20241109_Log_Fireflyiii.txt)

 Are there any additional logs or configurations I can check to gather more details?"

**Reply by JC5** [reply]
https://docs.firefly-iii.org/how-to/general/debug/

Sorry no, without more logs there's little I can do.

**Comment by Tweed-Bear** [comment]
I've completed all the instructions on the page, but the logs are still not updating more. 

Thanks for your help, I'll investigate further.
